submission-scripts/script-templates/Makefile

```diff
@@ -704,3 +760,39 @@ define create-storm
 	sed -i -e "s/ZOOKEEPER_SETUP=.*/ZOOKEEPER_SETUP=yes/" $(MAGPIE_STORM)
 	sed -i -e "s/ZOOKEEPER_VERSION=\"\(.*\)\"/ZOOKEEPER_VERSION=\"$(STORM_ZOOKEEPER_VERSION_DEFAULT)\"/" $(MAGPIE_STORM)
 endef
+
+define create-tensorflow
+	echo "Creating magpie.$(SCHED)-tensorflow"
+	cat 	magpie-shebang \
+		magpie-header-llnl \
+		magpie-config-$(SCHED)-header \
+		magpie-config-$(SCHED)-distributed \
+		magpie-magpie-customizations-substitution-tensorflow \
+		magpie-general-configuration-header \
+		magpie-general-configuration-python \
+		magpie-tensorflow \
+		magpie-run-job-header-substitution \
+		magpie-run-job-$(DIST)-distributed > $(MAGPIE_TENSORFLOW)
+	$(call common-substitution, ${MAGPIE_TENSORFLOW},${JAVA_DEFAULT})
+	$(call common-additions, ${MAGPIE_TENSORFLOW}, tensorflow)
+	sed -i -e "s/MAGPIE_JOB_TYPE=\"\(.*\)\"/MAGPIE_JOB_TYPE=\"tensorflow\"/" $(MAGPIE_TENSORFLOW)
+	sed -i -e "s/TENSORFLOW_SETUP=.*/TENSORFLOW_SETUP=yes/" $(MAGPIE_TENSORFLOW)
+endef
+
+define create-tensorflow-horovod
+	echo "Creating magpie.$(SCHED)-tensorflow-horovod"
+	cat 	magpie-shebang \
+		magpie-header-intel \
```

Comment:
b/c it also includes llnl stuff, I think you need to include the llnl header too

Reply:
of course, my oversight, sorry